src/main/resources/application-test.properties

```diff
@@ -0,0 +1,9 @@
+spring.datasource.url=jdbc:h2:mem:db;MODE=PostgreSQL;DATABASE_TO_LOWER=TRUE;DEFAULT_NULL_ORDERING=HIGH
+spring.datasource.driver-class-name=org.h2.Driver
+
+spring.jpa.hibernate.ddl-auto=update
+
+fixtures.users.test1.email=${TEST1_EMAIL}
```

Comment:
I created actual gmail accounts for these users. Setting up strong testing infrastructure is a nice to have.